MANIFEST.in

```diff
@@ -1,23 +1,14 @@
-include COPYING.md
-include CONTRIBUTING.md
-include README.md
+include README.md LICENSE.md CONTRIBUTING.md requirements.txt requirements-test.txt requirements-doc.txt
 
 # Documentation
-graft docs
```

Comment:
Why it is need to include documentation files into python package? Classes and methods already have docstrings included. Only CLI tools help here can be useful, but it is not available for `man` command, so it's easier to open online docs instead.

Reply:
Docs are typically grafted so that the docs sources are preserved in the sdist. It is mainly for archival purposes. Opinions are split on whether this is a good thing or not.

Reply:
IMHO, package builds should never be used for archival purpose. This should be done by adding tags into the repo.

Reply:
This is also for linux distributions that refuse to build from commit and build from sdist. It's less relevant with everything connected, but if you've ever worked on an air gapped system you are in general supper happy to have the docs as part of the package. 

Same reasoning for test suite can be made. 

I don't have any objections though, as Jupyter_client is rarely user facing, though for other packages I think it is good to include it.

Reply:
Ok, I've reverted this change